README.rst

```diff
@@ -539,6 +539,10 @@ Choose from the list of available rules:
   - ``keep_multiple_spaces_after_comma`` (``bool``): whether keep multiple spaces
     after comma; defaults to ``false``
 
+* **method_chaining_indentation**
+
+  Method chaining MUST be indented with one tab.
```

Comment:
I would not use the word *tab* as code can be indented with spaces (and that's what most PHP projects do nowadays). What about:

```
Method chaining MUST be properly indented.
```